### PR TITLE
github actions compile fix

### PR DESCRIPTION
**Pull request by gretel**
kinda sorta self explanatory :)

### CODE REVIEW

**Comment by gretel** [comment]
the artifacts resulting on the action is what would be put on bintray. meaning this makes bintray superflous.

**Comment by nerdCopter** [comment]
I agree.
Only issue is Test-Pilots have zero access to Drafts.  Thus the reason for Bintray.

**Comment by gretel** [comment]
that is not a reason for bintray. `draft` is a attribute per deploy which can be toggled. it's just not such a good idea to do this for initial testing.
furthermore, what is keeping you bintray lovers from using the according github action?!
as i said before, it's super waste of resource to run travis which fails often just to put binaries up on bintray.
https://github.com/marketplace/actions/action-upload-bintray and others.

**Comment by nerdCopter** [comment]
i don't love bintray. i hate bintray. 
"we" need an alternative to allow test-pilots access to builds we wish to share, yet keep relatively private and "unreleased".   but i digress.  i'm not wanting conflict of interest.

**Comment by gretel** [comment]
i brought up an alternative months ago.
what are you talking about. closed this due to frustration

**Comment by nerdCopter** [comment]
no need to close due frustration. i'm certainly not against anything. we are open to solutions.  if this is a proper solution then awesome and thank you.  let the team members review.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,20 +1,63 @@
...
+
+    - name: Release Binaries
+      uses: softprops/action-gh-release@v1
+      if: startsWith(github.ref, 'refs/tags/')
+      with:
+        prerelease: true
+        draft: true
+        files: obj/*.hex
+      env:
+        GITHUB_TOKEN: ${{ secrets.release_token }}
```
Comment (gretel) [inline .github/workflows/build.yml:63]: due to current limitations of github the token is bound to my account. it should be able to issue token via a organizations in the future. any valid token by a orga member having admin priviliges will do.

**Comment by gretel** [comment]
adjusted github settings accordingly so merges depend on github action builds, not travis. if this works not out as intented feel free to revert. but this should resume approval workflow now.

**Comment by gretel** [comment]
somebody needs to issue a token asap and use it cause i can't sustain working on this.

**Comment by nerdCopter** [comment]
Welcome to leave it un-maintained if you wish.
The existing token works as-is, but if you choose to delete it, then we will create a new one.  Thank you for all your efforts and work thus far.  It is well appreciated.

**Pull request merged**